handler/src/main/java/io/netty/handler/ssl/OpenSslContext.java

```diff
@@ -507,73 +497,74 @@ static long toBIO(PrivateKey key) throws Exception {
         if (key == null) {
             return 0;
         }
-        ByteBuf buffer = Unpooled.directBuffer();
+
+        ByteBufAllocator allocator = ByteBufAllocator.DEFAULT;
```

Comment:
nit: just pass `ByteBufAllocator.DEFAULT` directly ... no need for temp variable (even if we think JDK may optimize it away)